src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -369,6 +369,13 @@ private SdkResult ResolveSdkUsingAllResolvers(int submissionId, SdkReference sdk
                     sdkResult = result;
                     return true;
                 }
+                else
+                {
+                    string resultWarningsAndErrors = (result.Warnings?.Any() == true ? "\nWarnings:\n" + string.Join("\n", result.Warnings) : string.Empty)
```

Comment:
`Warnings` and `Errors` needs to be translated even for low priority messages.

Reply:
Thanks!
I included `Warnings` and `Errors` in the message so they can be translated. Example of a new message: 
```
The "SdkResolver" resolver attempted to resolve the SDK "SDK".
Warnings: null
Errors: Error1
Error2
```